server/lib/amplitude.js

```diff
@@ -38,19 +38,7 @@ const GROUPS = {
   sms: 'fxa_sms'
 };
 
-const ENGAGE_SUBMIT_EVENT_GROUPS = {
```

Comment:
A happy side effect of this fix is that the categorisation for `engage` and `submit` events is the same as `view` events again, so we can use the same lookup for both.